intranet/static/js/announcement.form.js

```diff
@@ -24,6 +24,12 @@ $(function() {
             }
         }
 
+        if($("#id_title").val().match(/\bION\b/) || editor.getData().match(/\bION\b/)) {
```

Comment:
You should probably add a comment explaining what this is doing (for future lookers) and link to the appropriate issue.